server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolPreprocessor.g4

```diff
@@ -24,6 +24,11 @@ includeStatement
    :EXEC_SQL INCLUDE copySource END_EXEC DOT_FS
    ;
 
+// copy maid statement
+copyMaidStatement
+    : LEVEL_NUMBER? COPY MAID copySource DOT_FS?
```

Comment:
In case `COPY MAID` works with replacement you can just modify the `copyStatement` and add optional `MAID?` there.

Reply:
Thank you @grianbrcom . I am not aware of any additional syntax like replacement or else. We will see once the customer tries it out and gets back to us with their results.